_opensearch/bucket-agg.md

```diff
@@ -74,6 +74,98 @@ The `terms` aggregation requests each shard for its top 3 unique terms. The coor
 
 This is especially true if `size` is set to a low number. Because the default size is 10, an error is unlikely to happen. If you don’t need high accuracy and want to increase the performance, you can reduce the size.
 
+### Account for pre-aggregated data
+
+While the `doc_count` field provides a representation of the number of individual documents aggregated in a bucket, the field by itself does not have a way to account for documents that store pre-aggregated data, such as `histogram`. To account for pre-aggregated data and accurately calculate the number of documents in a bucket, you can use the `_doc_count` field to add the number of documents in a single summary field. When a document includes the `_doc_count` field, all bucket aggregations recognize its value and increase the bucket `doc_count` proportionately. Keep these considerations in mind when using the `_doc_count` field:
```

Comment:
Can we rephrase this to simplify?
While the `doc_count` field represents the number of individual documents aggregated in a bucket, the field by itself does not account for documents that store pre-aggregated data, such as `histogram`.

Reply:
Also,
When a document includes the `_doc_count` field, all bucket aggregations increase the bucket `doc_count` by the value of `_doc_count`.